VimR/VimR/AppearancePref.swift

```diff
@@ -294,8 +273,8 @@ class AppearancePref: PrefPane, NSComboBoxDelegate, NSControlTextEditingDelegate
   }
 
   private func updateViews() {
-    if let familyName = self.font.familyName { self.fontPopup.selectItem(withTitle: familyName) }
-    self.sizeCombo.stringValue = String(Int(self.font.pointSize))
+    sharedFontPanel.setPanelFont(font, isMultiple: false)
+    self.fontPanelButton.title = font.displayName.map { "\($0) \(font.pointSize)" } ?? "Show fonts..."
```

Comment:
is the styling, text etc of this button acceptable?

<img  alt="Screenshot 2020-04-20 at 22 26 46" src="https://user-images.githubusercontent.com/631757/79801568-3a7e4000-8356-11ea-8c84-7e1d2c796c6e.png">

Reply:
👍